### PR TITLE
Button Page Content and Media Updates

### PR DESCRIPTION
**Pull request by diriquelme**
## What does this PR do?
Updated documentation and added images for buttons.

### Associated Issue
#3966 and #3926

### Documentation
 - [x] Update Spark Docs

### Deploy Preview Reviewed and Approved
 - [ ] Design Team
 - [ ] Accessibility Team

### CODE REVIEW

**Comment by diriquelme** [comment]
Updated images and content and added anchor links to "States" sections

File: src/pages/using-spark/components/button.mdx [hunk 1]
```diff
@@ -198,28 +274,118 @@ Use when data is being saved or submitted. The spinning indicator will replace t
...
+  alt="On a mobile screen the main action is on top of the secondary action."
+/>
+
+<SprkDivider element="span" additionalClasses="sprk-u-Measure"></SprkDivider>
+
 ## Anatomy
 
-- A Button must contain text and/or an Icon
+- A Button must have a text label and/or icon, and a container.
+- A Button's Text communicates the action that will happen when the
```
Comment (bobwise) [inline src/pages/using-spark/components/button.mdx:379]:
```suggestion
- A Button's text communicates the action that will happen when the
```

**Review by bobwise — CHANGES_REQUESTED** [review]
Great job on this @diriquelme! All these new guidelines and examples are going to be very useful for our users. I left a few comments and made some small grammar/spelling suggestions.

File: src/pages/using-spark/components/button.mdx [hunk 2]
```diff
@@ -1,13 +1,38 @@
...
+export const fullWidthButtonSet = `${fullWidthButton} 560w, ${fullWidthButton} 1120w`;
+import stackedButtons from '../../../images/buttons/stackedButtons.png';
+import stackedButtons2x from '../../../images/buttons/stackedButtons@2x.png';
+export const stackedButtonsSet = `${stackedButtons} 560w, ${stackedButtons} 1120w`;
+import sideBySide from '../../../images/buttons/sideBySide.png';
+import sideBySide2x from '../../../images/buttons/sideBySide@2x.png';
+export const sideBySideSet = `${sideBySide} 560w, ${sideBySide} 1120w`;
+import verticalButtons from '../../../images/buttons/verticalButtons.png';
+import verticalButtons2x from '../../../images/buttons/verticalButtons@2x.png';
+export const verticalButtonsSet = `${verticalButtons} 560w, ${verticalButtons} 1120w`;
```
[the remark this answers is not in the document]
Comment (diriquelme) [reply src/pages/using-spark/components/button.mdx:30]: Thank you for reviewing it.

**Review by diriquelme — COMMENTED** [review]
All Bob changes look good and are ready to be approved.

**Comment by diriquelme** [comment]
Please approve all @bobwise edits, thank you!

**Review by bobwise — APPROVED** [review]

File: src/pages/using-spark/components/button.mdx [hunk 3]
```diff
@@ -94,18 +82,40 @@ Represents the main call to action on the page. Use no more than two Primary But
...
-- Back
-- In a multi-step form, a Secondary Button works with Primary and Tertiary Button as a set of related actions.
-  - Primary Button is "Submit"
-  - **Secondary Button is "Back"**
-  - Tertiary Button is "Cancel"
+<img
+  srcSet={buttonModalSet}
+  class="sprk-u-Measure sprk-u-pam sprk-u-pbm sprk-u-Width-100"
+  src={buttonModal}
+  alt="A primary button is on the right and represents the main action"
```
Comment (guern1kn) [inline src/pages/using-spark/components/button.mdx:97]: This alt text needs a period at the end.

File: src/pages/using-spark/components/button.mdx [hunk 4]
```diff
@@ -132,15 +145,65 @@ Represents less prominent actions. Sometimes in an action set with [Primary](#pr
...
+- Use for sub-tasks where a Primary (Filled) button is the main action.
+- In a button group, use Tertiary buttons for actions with the least visual emphasis that do not require the user's attention.
+
+**Example:**
+
+<img
+  srcSet={buttonFormSet}
+  class="sprk-u-Measure sprk-u-pam sprk-u-pbm sprk-u-Width-100"
+  src={buttonForm}
+  alt="The back and cancel button represent least important actions in the button group"
```
Comment (guern1kn) [inline src/pages/using-spark/components/button.mdx:160]: This alt text needs a period at the end.

File: src/pages/using-spark/components/button.mdx [hunk 5]
```diff
@@ -198,28 +274,118 @@ Use when data is being saved or submitted. The spinning indicator will replace t
...
+#### Button Alignment
+
+As a rule, place the (Primary) button **on the right side** of a screen
+to communicate the main action and capture's the user's attention.
+
+> This rule follows the Gutenberg Principle, which describes how a
+> user's eye makes a z-pattern when scanning a screen and implies that
+> placing buttons to the right communicates the main action to the user.
+
+- Buttons should be **right aligned** to its parent container.
```
Comment (guern1kn) [inline src/pages/using-spark/components/button.mdx:331]: it's

File: src/pages/using-spark/components/button.mdx [hunk 6]
```diff
@@ -198,28 +274,118 @@ Use when data is being saved or submitted. The spinning indicator will replace t
...
+- Grouped buttons should be **16px** away from each other.
+- Apply the same width to all buttons in a group.
+- Do not use buttons in paragraphs or within lines of text. Use
+  [Link](/using-spark/components/link)
+  instead.
+
+#### Button Alignment
+
+As a rule, place the (Primary) button **on the right side** of a screen
+to communicate the main action and capture's the user's attention.
```
Comment (guern1kn) [inline src/pages/using-spark/components/button.mdx:325]: captures

File: src/pages/using-spark/components/button.mdx (hunk 3 above)
Comment (diriquelme) [reply src/pages/using-spark/components/button.mdx]: Added and also capitalized "Primary Button"

File: src/pages/using-spark/components/button.mdx [hunk 7]
```diff
@@ -198,28 +274,118 @@ Use when data is being saved or submitted. The spinning indicator will replace t
...
-  componentName="button--full-width-at-small-viewport"
-  hasReact
-  hasAngular
-  hasHTML
-  titleAttr="Full Width at Smaller Viewports Button Example"
+**Exception:**
+
+- For layouts that require more than three calls to action in a group.
+  - Use one Primary button for the main action in combination with two less visual weight buttons.
+  - If using a primary button in the group, prefer all other buttons to be tertiary.
```
[the remark this answers is not in the document]
Comment (diriquelme) [reply src/pages/using-spark/components/button.mdx]: Capitalized :)

**Review by guern1kn — APPROVED** [review]
Nice work on this!

**Pull request merged**